src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/MouseEventSender.cs

```diff
@@ -366,11 +365,10 @@ public static void SendMouseEvent(int replaySegment, MouseInputActionData mouseI
             }
 
             SendRawPositionMouseEvent(replaySegment, normalizedPosition, mouseInput.leftButton, mouseInput.middleButton, mouseInput.rightButton, mouseInput.forwardButton, mouseInput.backButton, mouseInput.scroll);
-
         }
 
 #if ENABLE_LEGACY_INPUT_MANAGER
-        private static void SendMouseEventLegacy(Vector2 position, Vector2 delta, Vector2 scroll,
+        public static void SendMouseEventLegacy(Vector2 position, Vector2 delta, Vector2 scroll,
```

Comment:
This should stay private, it is called internally from sendrawposition mouseevent